Migrate Headings components into component package (#79)
BBC-archive/psammead: packages/components/psammead-headings/README.md

# psammead-headings &middot; [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/BBC-News/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/psammead-headings.svg)](https://www.npmjs.com/package/@bbc/psammead-headings) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://reactjs.org/docs/how-to-contribute.html#your-first-pull-request)

## Description

> The Headings are a set of two components, `Headline` and `Subheading`. They use a `h1` and `h2` HTML element respectively, with the `Headline` being designed for a single use on the page, with `Subheading` being aimed at repeated use.

Reviewer: ```suggestion
The Headings are a set of two components, `Headline` and `SubHeading`. They use a `h1` and `h2` HTML element respectively, with the `Headline` being designed for a single use on the page, with `SubHeading` being aimed at repeated use.
```

## When to use this component

> These components can be used at any point on the page, however the `Headline` is designed to be used once at the top of the page. The `Subheadline` adds an `id` value to the `h2` which can be used as an anchor when referencing content.

Reviewer: ```suggestion
These components can be used at any point on the page, however the `Headline` is designed to be used once at the top of the page. The `SubHeading` adds an `id` value to the `h2` which can be used as an anchor when referencing content.
```

## Accessibility notes

> The `SubHeadline` component has a tabindex of `-1` which removes the element from the default navigation flow, meaning the user can not tab to it.

Reviewer: I don't think this is the reason we have a tabindex.

```suggestion
The `SubHeading` component has a tabindex of `-1` so that it works correctly with screen readers when navigated to via a skip link.
```
